packages/tenant-process/src/services/tenantService.ts

```diff
@@ -556,6 +561,86 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async revokeVerifiedAttribute({
+      tenantId,
+      attributeId,
+      organizationId,
+      correlationId,
+    }: {
+      tenantId: TenantId;
+      attributeId: AttributeId;
+      organizationId: TenantId;
+      correlationId: string;
+    }): Promise<Tenant> {
+      logger.info(
+        `Revoke verified attribute ${attributeId} to tenant ${tenantId}`
+      );
+
+      if (organizationId === tenantId) {
+        throw verifiedAttributeSelfRevocation();
+      }
+
+      await assertAttributeRevocationAllowed({
+        producerId: organizationId,
+        consumerId: tenantId,
+        attributeId,
+        readModelService,
+      });
+
+      const targetTenant = await retrieveTenant(tenantId, readModelService);
+
+      const verifiedTenantAttribute = targetTenant.data.attributes.find(
+        (attr) =>
+          attr.type === tenantAttributeType.VERIFIED && attr.id === attributeId
+      ) as VerifiedTenantAttribute;
+
+      if (!verifiedTenantAttribute) {
+        throw attributeNotFound(attributeId);
+      }
+
+      const verifier = verifiedTenantAttribute.verifiedBy.find(
+        (a) => a.id === organizationId
+      );
+
+      if (!verifier) {
+        throw attributeRevocationNotAllowed(tenantId, attributeId);
+      }
+
+      const revoker = verifiedTenantAttribute.revokedBy.find(
+        (a) => a.id === organizationId
+      );
+
+      if (revoker) {
+        throw attributeAlreadyRevoked(tenantId, organizationId, attributeId);
+      }
```

Comment:
```suggestion
      if (
        verifiedTenantAttribute.revokedBy.some((a) => a.id === organizationId)
      ) {
        throw attributeAlreadyRevoked(tenantId, organizationId, attributeId);
      }
```
Since we don't need a const for `verifier` we can refactor this check like this